sdk/cloudmachine/Azure.Provisioning.CloudMachine/src/FeaturesBuiltIn/ServiceBusNamespaceFeature.cs

```diff
@@ -37,7 +37,9 @@ protected override ProvisionableResource EmitCore(CloudMachineInfrastructure inf
                 (ServiceBusBuiltInRole.GetBuiltInRoleName(ServiceBusBuiltInRole.AzureServiceBusDataOwner), ServiceBusBuiltInRole.AzureServiceBusDataOwner.ToString()),
             ]);
 
-        Emitted = _serviceBusNamespace;
         return _serviceBusNamespace;
     }
+
+    protected internal override void EmitConnections(ConnectionCollection connections, string cmId)
+        => connections.Add(CloudMachineConnections.CreateDefaultServiceBusConnection(cmId));
```

Comment:
Possibly, but I have to say I don't know what the format of these SB URIs is. I can change it to use name, and it will work because today the id == name, but before we make SB feature public and truly configurable, I won't be able to test this change.